src/main/java/com/skytask/channel/ProductSource.java

```diff
@@ -0,0 +1,18 @@
+package com.skytask.channel;
+
+import org.springframework.cloud.stream.annotation.Input;
+import org.springframework.cloud.stream.annotation.Output;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.SubscribableChannel;
+
+public interface ProductSource {
+
+    @Output("createProductChannel")
```

Comment:
Can't we put Output's value on constants?

Reply:
Yes, in this PR I didn't have that, but now I have them as constants.